### PR TITLE
LPS-129397

### PR DESCRIPTION
**Pull request by martamedio**
cc/ @csierra and @stian-sigvartsen

### CODE REVIEW

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -19,6 +19,9 @@ are-you-sure-you-want-to-delete-the-application-this-action-revokes-one-authoriz
 are-you-sure-you-want-to-delete-the-application-this-action-revokes-x-authorizations-and-associated-tokens=Are you sure you want to delete the application? This actions revokes {0} authorizations and associated tokens.
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties:19]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by martamedio** [comment]
ci:test:sf

**Comment by csierra** [comment]
LGTM @martamedio

**Comment by martamedio** [comment]
ci:forward

**Pull request closed without merging**